base/client.jl

```diff
@@ -55,6 +57,8 @@ warn_color()   = repl_color("JULIA_WARN_COLOR", default_color_warn)
 info_color()   = repl_color("JULIA_INFO_COLOR", default_color_info)
 input_color()  = text_colors[repl_color("JULIA_INPUT_COLOR", default_color_input)]
 answer_color() = text_colors[repl_color("JULIA_ANSWER_COLOR", default_color_answer)]
+bt_linfo_color()   = repl_color("JULIA_BT_LINFO_COLOR", :bold)
+bt_funcdef_color() = repl_color("JULIA_BT_FUNCTION_COLOR", :bold)
```

Comment:
these abbreviations aren't very readable, perhaps just spell them out? Also `LambdaInfo` and `Function` no longer exist (replaced by method signature and method name, respectively)`